lib/static/css/main.css

```diff
@@ -307,6 +307,13 @@ header h2 {
 .mainContainer .wrapper .post.basicPost {
   margin-top: 30px;
 }
+
+.mainContainer .wrapper .post h1 {
+  margin-top: 20px;
+  margin-bottom: 0;
+  font-size: 250%;
```

Comment:
Just curious how you chose this value, 250%? I think it is fine, just wondering what you based it upon.

Reply:
It's based on the size of the original H1 element on the page (based on the title key)